cmd/cmd-mewn.go

```diff
@@ -5,7 +5,7 @@ package cmd
+  mewn.AddAsset(".", "../runtime/js/runtime/init.js", "1f8b08000000000000ff548ccd4ec3301084cfec53cc3189680c1c90a04282231247242e488b9b6e89a96347b64b404ddf1dd96d0ff867359a99fd5443601c0f9f049f14cd988ba1a098195c715d149744cd39c8dd0f7552ac72a2f2cb3b47bbcab41a3433abfcdf992f59959b37007aed05d67cf669923cb1097a90c9872d363e6092d5c29aad408f63a4aaabf1221a4f2ef5defde2e6eafa6e310689e212358a5483afd81b9720f15b4234dedde3168d22524d4368f0ec4c32da9a2811a917bc696323c2ce253308a1741e471df480fd66e7ba64bc3ba0d3d6ae74b7a58c3adb85559da31a7bba988c5bfba99d32b4e5f67f614907a2c1af77565af9197d48110f05b2a4bf000000ffffe38437ed8a010000")
+  mewn.AddAsset(".", "./linuxdb.yaml", "1f8b08000000000000ffec584b6fdc3610bee757cc690bd8a0ddda3de916d46d0e8d9ba0467ae8a5a1a83197111f021f8eb7e88f2fb89245adb54f2d17db02f1c93312bf8f1fe7b1431142de54c2792bcae085d1ae7803506129a88eff0188aae8eca569512275e8da87f1d5471aa47f3101345558c05d5a11ff9ed03a6174f1f27aff8033d63d634629aaab02669cb12aa8a6f3031918fdbac61a6ea97205ccdacd75c02ffefe4500d26d893336f002cc5136057c5cca01a19da752c257e1e7f0d985ca006d3ce1e8fb476510b222e81c6a2fa8fc0c5457e0ed0228a742af216c6a4e98d18f821fc59b607653ea46edcbc5829540dc7b987bdfb8e2fabac2f24a9b0a9d0996e11533eadaa10fcd5f3fdc5c3dc33fb0dc5c49dd1c08cc66b076af640111e28bdbb455294a4badc051e0fa076b544951725f935b52e1d351473904da7d9852945fb1ac85bf898b7ebcfa3e07ff1871bc915006ed432abfd6deaffc36565aabe95382da517e8f41ca95121c38d694e1c5f6321cc4fd6263dc6b2a45521dad2c9a7fa552c0bbdf3e5dbf173a3c6fd37e31d67e22a90db5d6f824b6b5b3c8fdb8843a4cf069c5fe6dac18fc9a2ccd2c52ffec91ceaa4fc66356420fe2d9bbb2e85cc611ee5fe0ce2a16252ad49eda45529b7c59e42638f8f0f0dfa8578d6690c1d1cad39aee7e4e6067175921366265ec8b7616a17709eaec329f8ca892c8686591f88711d5f10df7b8c1f5b96c1ce90756ea300e187892597585ea4433ea0ac7be83e5ba13f4f5e56d7b1493a807eb77eb1b4c78d3195f838c69196a6f5ccae3d6ce92c93fa1f61f1ee0dcc9bc1ca11741f56ece18619797a0688d27bb7e0d099b9a470472a2f41ef161839274d1ebef58c3178eab81034b607cfe3dc4be559081f435d098fa112b6369aa84d6ce5209bf24a81d35b0f567eb5b1d6ce01b53fddf32fc265b8adf6c23a796cd5386472b4b7ebfb56c9ea1cfc32cee28ff27b8863245359087b872af4c8fd93991a25b9dbfc3270adda87d3e8c0d0e73db67b1982f538fd3d7b78764f8449a04b036a3cdf296bc92d6ad2b576e9b1c1f40b6a5f6f0beb1216a8aea2fd49aa4b27364d178df62e528e14c3a09b56aa4353a73ea256f7fbf3faf5a1ea7ed41505b3b8bc677096aaabca9ad3738a1392c4cb0e016cea3facec56aae29c7184acad11e39604c63d8abe31e02bdf3b6581cd65951a1e5d8addcb7b192431b6bc79256bf62fa370000ffff57890681cc1b0000")
```

Comment:
I honestly have no idea why is this happening.
I deleted everything, forked & cloned anew. 
when I run `mewn` within `wails/cmd` the generated file is this.

please advise.

Reply:
No idea. Hack would be to copy the new yaml file line and Pate over old one. Let's just remove embedding this file altogether. I can do that if you like.

Reply:
I'll give it a try :+1: